src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -1,19 +1,52 @@
 package core.basesyntax.impl;
 
 import core.basesyntax.Storage;
+import java.util.Objects;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_SIZE = 10;
+    private final Object[] keys;
+    private final Object[] values;
```

Comment:
lets use our generic types
```suggestion
    private final K[] keys;
    private final V[] values;
```

Reply:
+